promql/engine.go

```diff
@@ -376,7 +386,11 @@ func (ng *Engine) exec(ctx context.Context, q *query) (model.Value, error) {
 	const env = "query execution"
 
 	evalTimer := q.stats.GetTimer(stats.TotalEvalTime).Start()
-	defer evalTimer.Stop()
+	defer func() {
+		evalTimer.Stop()
+		queryTimingStats.WithLabelValues("eval_total").
```

Comment:
The label name is rather unfortunate, but I wanted to keep consistent with the names of the existing internal stats.
Although the label name has "total" in it, it's actually just the timing of a step in the query evaluation, as far as I understood the code.
Can you think of a better name for it? I don't have enough insight into the history of these stats and their names.